mate-scripts/src/localScripts/repositoriesData.json

```diff
@@ -0,0 +1,173 @@
+{
```

Comment:
looks like dynamic data. do we need to store it?

Reply:
not fully dynamic, better to leave as is